src/compiler/emitter.ts

```diff
@@ -4785,8 +4785,10 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
 
                 // emit name if
                 // - node has a name
-                // - this is default export and target is not ES6 (for ES6 `export default` does not need to be compiled downlevel)                
-                if ((node.name || (node.flags & NodeFlags.Default && languageVersion < ScriptTarget.ES6)) && !thisNodeIsDecorated) {
+                // - this is default export and target is not ES6 (for ES6 `export default` does not need to be compiled downlevel)
+                // - this is default export with static initializers
+                if ((node.name || (node.flags & NodeFlags.Default && (languageVersion < ScriptTarget.ES6

```

Comment:
It looks like the previous version may have been a vestige of our class emit before we split the function.